README.md

```diff
@@ -1,16 +1,33 @@
 # media-ui-extensions
 Extending the HTMLVideoElement API (`<video>`) to support advanced video player user-interface features
 
-With the addition of [web components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) to the browser, video player developers can create custom elements that mimic the video tag API, with goals of creating stand-in compatiblity with the video tag and compatibility across players. The video tag API is however lacking some important functions to support modern player UIs, including playback quality/resolution selection and awareness of ads. This repo is intended to capture requests and proposals for those functions.
+With the addition of [Web Components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) to the browser, video player developers can create custom elements that mimic the video tag API, with goals of creating stand-in compatibility with the video tag and compatibility across players. The video tag API is however lacking some important functions to support modern player UIs, including playback quality/resolution selection and awareness of ads. This repo is intended to capture requests and proposals for those functions.
 
-## Submitting a proposal
-Use `template.md` to frame your proposal and make a pull-request to add your proposal document to the `proposals` directory.
+## Goals for Proposals
+we want these proposals to be something that we can eventually propose to the Media WG or WhatWG as additional features to the video element and anything related. In the meantime can be used as a spec to keep things interoperable between implementers.
+
+## Before submitting a proposal
+If you have a new idea, it might be worth creating an issue to discuss it a bit first before submitting a proposal to make sure that it's something could be a good fit and won't be less likely to be rejected later on.
 
-Inspired by [rust-lang/rfcs](https://github.com/rust-lang/rfcs) via [video-dev/hlsjs-rfcs](https://github.com/video-dev/hlsjs-rfcs).
+## Submitting a proposal
+- Fork the repo
+- Copy `0000-template.md` to `proposals/0000-your-feature.md`. Don't change the number yet as that will be updated when the PR is created.
+- Fill out your proposal with as much detail as possible.
+- Submit a pull request. Once the PR is created, the file `0000-` prefix and the header of the file should be updated to point at the new PR number.
+- Folks should now review the proposal and suggest changes. While in the PR, it's expected that many changes may be made. Ideally, please don't rebase commits in the branch as it'll make reviewing only new changes easier.
+- At some point, a team member should issue a Call for Consensus (CfC). This CfC will include whether this proposal should be accepted, rejected, or postponed.
+- The CfC will last a minimum of 2 weeks to make sure that everyone gets a chance to review
```

Comment:
suggestion(non-blocking): do we want to make it a hard minimum of 2 weeks, or should we allow for something more like, 

> The CfC will last a minimum of 2 weeks to make sure that everyone gets a chance to review. If all relevant parties have explicitly finished reviewing before 2 weeks, the CfC may end early.

This may depend on who counts as "everyone" here. If we want to treat this more as an open "request for comments" to the larger community, we'd likely want a clear, fixed time window. If we're instead more concerned about "explicit relevant parties" (which could vary per proposal), my suggested change may be ok.